src/i18n/index.ts

```diff
@@ -0,0 +1,61 @@
+import { langDe, langEn, langEs } from './langResources';
+
+import { ILangResource, ILangResourceCollection, IOSSpecific, LangResourceKeys } from '../models/i18n';
+
+const langResourceCollection: ILangResourceCollection = {
+  de: langDe,
+  en: langEn,
+  es: langEs,
+};
+
+export class LanguageResourceManager {
+
+  constructor(private resourceCollection?: ILangResourceCollection | {}) {
```

Comment:
In order to avoid passing the `language` on each `getMessage` call we could set the default language in the constructor and make the `language` parameter  optional in the `getMessage` method, just in case somebody would like to retrieve a specific translation.

I think this would help to reduce the boilerplate in all the translations while keeping the same functionality that we have right now.